bundle.proto

```diff
@@ -14,3 +14,41 @@ message BundleUuid {
   bundle.Bundle bundle = 1;
   string uuid = 2;
 }
+
+/* Bundle Result Types */
+
+// Indicates the bundle was forwarded to validators.
+message Forwarded {
+  // Maps slots to validator pubkeys this bundle was forwarded to.
+  map<uint64, string> validator_identities = 1;
```

Comment:
wdyt about this being just one validator + slot? if we send to multiple we can emit multiple events?

Reply:
in practice when the system has wide adoption we should only really be sending to one validator unless we're close to boundary